todo_app/app.py

```diff
@@ -3,23 +3,30 @@
 from todo_app.enums import Status
 
 from todo_app.flask_config import Config
+from todo_app.models import ViewModel
 
-app = Flask(__name__)
-app.config.from_object(Config())
-trello_service = TrelloService()
+def create_app():
+    app = Flask(__name__)
+    app.config.from_object(Config())
+    trello_service = TrelloService()
 
-@app.route('/', methods = ['GET'])
-def index():
-    items = trello_service.get_items()
-    return render_template('index.html', items = items, incomplete=Status.INCOMPLETE)
+    @app.route('/', methods = ['GET'])
+    def index():
+        items = trello_service.get_items()
+        item_view_model = ViewModel(items)
+        return render_template('index.html', view_model=item_view_model, incomplete=Status.INCOMPLETE)
 
-@app.route('/', methods = ['POST'])
-def add_new_item():
-    item = request.form['item']
-    trello_service.add_item(item)
-    return redirect('/')
+    @app.route('/', methods = ['POST'])
+    def add_new_item():
+        item = request.form['item']
+        trello_service.add_item(item)
+        return redirect('/')
 
-@app.route('/complete/<item_id>', methods=['POST'])
-def complete_item(item_id):
-    trello_service.move_to_complete_column(item_id)
-    return redirect('/')
\ No newline at end of file
+    @app.route('/complete/<item_id>', methods=['POST'])
+    def complete_item(item_id):
+        trello_service.move_to_complete_column(item_id)
+        return redirect('/')
+    
+    return app
+
+app = create_app()
```

Comment:
We don't need this final line, and actually it breaks your integration test when run from the command line! The main motive for introducing this `create_app` function is so that we can delay when it gets executed and the `app` variable actually gets created for the sake of testing, and thankfully Flask is happy to just call the `create_app` function itself if it sees it, so no harm in just getting rid of this line

You should be able to see this error if you try running the tests with `poetry run pytest`, but frustratingly VSCode will actually load the _real_ .env file automatically when the tests run, so if you run through that interface the tests will actually pass anyway because:
* VSCode loads the real variables
* That enables this line to execute happily
* The tests override the environment variables
* The tests create a new version of the app, which uses the right variables

```suggestion
```